scripts/prevalence_helper.sh

```diff
@@ -83,6 +93,9 @@ echo "Committing the files and submitting an auto-merge pull request"
 if [[ `git status --porcelain` ]]; then
   TODAY=$(date +%Y-%m-%d)
 
+  git config --global user.name "Jeremy Blanchard"
```

Comment:
This should probably set in the github automation config rather than here (in case anyone runs this manually in the future). Also, consider creating an automation gmail account to attribute auto rolls (automation@microcovid.org?)

Reply:
Yeah. The difficulty with doing a separate username here is that the commit gets attributed to the "bot" user and then the merge gets attributed to me because of my personal access token. Which is just unnecessary. 

I agree this is not a great solution, though. I'd suggest we commit it as-is and I'll open a new issue to move it over to "fully bot mode" at some point. Would that work?

Reply:
Okay I opened a separate issue  (#1039) to address this and I'm gonna merge it as-is for now so we can build on it.